workspaces/js-x-ray/src/probes/isUnsafeCommand.ts

```diff
@@ -25,6 +25,17 @@ function isSpawnOrExec(
     name === "execSync";
 }
 
+function getCommand(commandArg) {
+  switch (commandArg.type) {
+    case "Literal":
+      return commandArg.value;
+    case "TemplateLiteral":
+      return commandArg.quasis.at(0).value.raw;
+    default:
+      throw new Error("Unhandled arguments");
+  }
+}
```

Comment:
Not sure about the error here, if that's already narrowed before then for me commandArg must be typed to ESTree.Literal<string> | ESTree.TemplateLiteral<string>